api-gateway/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
   implementation 'org.springframework.boot:spring-boot-starter-webflux'
   implementation 'org.springframework.boot:spring-boot-starter-actuator'
   implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
+  implementation 'org.bouncycastle:bcpkix-jdk18on:1.74'
```

Comment:
Indirect dependencies should be declared as constraints. See https://github.com/department-of-veterans-affairs/abd-vro/blob/1ab3968ce669af1de76d8c7238de8d4e1b1cfed2/gradle-plugins/src/main/groovy/shared.java.vro-dep-constraints.gradle#L27